src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1574,7 +1574,7 @@ internal enum ErrorCode
         ERR_ExpressionTreeContainsTupleBinOp = 8382,
         WRN_TupleBinopLiteralNameMismatch = 8383,
         ERR_TupleSizesMismatchForBinOps = 8384,
-        ERR_ExprCannotBeFixed = 9385,
+        ERR_ExprCannotBeFixed = 8385,
```

Comment:
Are there no references to `CS9385` in tests?

Reply:
I don't typically update all the textual references in VerifyDiagnostics (the comments). Only the error enum entry is really checked (for C# at least).